src/parse-balance-map.ts

```diff
@@ -48,10 +49,11 @@ export function parseBalanceMap(balances: OldFormat): MerkleDistributorInfo {
 
   // generate claims
   const claims = sortedAddresses.reduce<{
-    [address: string]: {amount: string; proof: string[]};
-  }>((memo, address) => {
+    [address: string]: {index: number; amount: string; proof: string[]};
+  }>((memo, address, index) => {
     const {amount} = dataByAddress[address];
     memo[address] = {
+      index,
```

Comment:
I'm not sure if this is the same index as in the contract code because there it depends on how the intermediate hash values are sorted. We might have to reconstruct it from the merkle proof itself, I can have a look at it later.